ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/TransactionServiceVisibilityIT.scala

```diff
@@ -130,6 +141,11 @@ class TransactionServiceVisibilityIT extends LedgerTestSuite {
     "TXNotDivulge",
     "Data should not be exposed to parties unrelated to a transaction",
     allocate(SingleParty, SingleParty),
+    tags = List(
+      securityAsset("Transactions").setHappyCase(
+        "Transactions are not exposed to parties unrelated to a transaction"
+      )
+    ),
```

Comment:
Not sure if we want to have a List here for ledger api tests... wdyt? Looks a bit too verbose